modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/upgrade-catch-all-check/LPD_48765.testjava

```diff
@@ -0,0 +1,19 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+/**
+ * @author Wilson Ferreira
+ */
+public class LPD_48765 {
+
+	public void method(AccountEntry accountEntry) {
+		if (accountEntry.isPersonalAccount()) {
+			System.out.println("test");
```

Comment:
Isn't a pattern of our code this type of thing. Please remove it.  
`return accountEntry;` shoud be fine to be instead.